features/mbedtls/inc/mbedtls/config.h

```diff
@@ -43,6 +43,12 @@
 #endif
 
 #else
+
```

Comment:
This part should be added in https://github.com/ARMmbed/mbed-os/blob/master/features/mbedtls/importer/adjust-config.sh#L56

Reply:
Will attempt, but may need help testing it.